niap-cc/Permissions/Tester/app/src/main/java/com/android/certifications/niap/permissions/config/TestConfiguration.java

```diff
@@ -58,7 +60,12 @@ default List<BasePermissionTester> getPermissionTesters(Activity activity) {
             permissionTesters.add(new PrivilegedPermissionTester(this, activity));
         }
         permissionTesters.add(new InternalPermissionTester(this, activity));
-        permissionTesters.add(new InstallPermissionTester(this, activity));
+        if(PermissionUtils.isNoPermissionManifest(activity.getApplicationContext())){
+            Log.i("TAG","No Permission Configuration should not include the Install Permission Tester");
```

Comment:
Why is the `InstallPermissionTester` not included in the no permission manifest? Since requested install permissions are granted when the app is installed, the no permission manifest is the only chance to verify that APIs and resources guarded by install permissions do not grant access when the permission is not granted.

Reply:
I separated install permission tester to 'InstallPermissionOnlyConfiguration' file in this case.
We can execute the install permission test cases with 
'RUN INSTALL PERMISSION TEST' button in the menu.  
So I think the user wouldn't miss to execute install permission tester,
As far as they are following our instructions.

As you know, No permission test cases raises a lot of errors and delays when execution.
(Some test cases need to await api timeouts.)
I observed inevitable anr error when I execute install tester and other testers at once.